src/utils/api.js

```diff
@@ -0,0 +1,43 @@
+const baseUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi';
+
+const appId = 'rQ2sGk73ALsSsV7Y7oHD';
+const appsEndpoint = `${baseUrl}/apps/`;
+const booksEndpoint = `${appsEndpoint}${appId}/books`;
+const deleteBookEndpoint = (id) => `${booksEndpoint}/${id}`;
+
+const postData = async (url, data, isText = false) => {
+  const res = await fetch(url, {
+    method: 'POST',
+    mode: 'cors',
+    headers: { 'Content-type': 'application/json' },
+    body: JSON.stringify(data),
+  });
+  if (isText) return res.text();
+  return res.json();
+};
```

Comment:
- [ ] When I add a book, it doesn't appear in the list of books immediately. I have to reload it before it appears in the list of books. I will like to advise you to remove `isText` parameter because it will always be `false` and the data you are receiving from the `API` endpoint is in `text` format. You need to return `res.text` and not `res.json`. Kindly implement this change 👍 

![chrome-capture](https://user-images.githubusercontent.com/77106962/162573723-6dfc7d4b-8d47-42cf-a122-cc583f931b56.gif)